lenny/routes/api.py

```diff
@@ -145,46 +268,49 @@ async def proxy_readium(request: Request, book_id: str, readium_path: str, forma
         return Response(content=r.content, media_type=content_type)
 
 
-@router.post('/items/{book_id}/borrow')
+@router.get('/items/{book_id}/borrow')
```

Comment:
We may also need a custom `options` handler for borrow to work with `Authorization` header.

e.g. 

```
@app.options("/v1/api/items/{item_id}/borrow")
async def options_borrow(item_id: str):
    return Response(status_code=204, headers={
        "Access-Control-Allow-Origin": "http://localhost:3000",
        "Access-Control-Allow-Methods": "GET",
        "Access-Control-Allow-Headers": "Authorization",
        "Access-Control-Allow-Credentials": "true",
    })
```